modules/sff/module/inc/sff/8636.h

```diff
@@ -149,6 +149,11 @@
 #define SFF8636_CC192_40GE_SWDM4         0x1F
 #define SFF8636_CC192_100GE_SWDM4        0x20
 #define SFF8636_CC192_100GE_PAM4_BIDI    0x21
+#define SFF8636_CC192_100GE_LR           0x27
```

Comment:
Please define them in increasing order of their value.

Reply:
addressed..